lib/fedex/tracking_information.rb

```diff
@@ -29,7 +29,7 @@ class TrackingInformation
     }
 
     attr_reader :tracking_number, :signature_name, :service_type, :status,
-                :delivery_at, :events, :unique_tracking_number
+                :delivery_at, :events, :unique_tracking_number, :estimated_delivery_at
```

Comment:
i told you this already but for the purposes of documentation:
i am replacing this line https://github.com/stitchfix/fashionthing/pull/980/files#diff-f7a570d3dfca86b346de504a175b2eceR33 with `tracking_data.estimated_delivery_at.to_date`